### PR TITLE
Update docsy theme to docsy v0.6.0

### PR DESCRIPTION
**Pull request by geriom**
Update the theme, the README file, and the hugo and go versions.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,6 +4,6 @@ go 1.12
 
 require (
 	github.com/FortAwesome/Font-Awesome v0.0.0-20220831210243-d3a7818c253f // indirect
```
Comment (deining) [inline go.mod:6]: This is not the latest version of Font-Awesome, this line should read:

```
	github.com/FortAwesome/Font-Awesome v0.0.0-20221115183454-96cafbd73ec4 // indirect
```

Use this command for upgrade:

```
hugo mod get -u github.com/FortAwesome/Font-Awesome@96cafbd73ec4
```

File: README.md [hunk 2]
```diff
@@ -11,10 +11,12 @@ In this project, the Docsy theme component is pulled in as a Hugo module, togeth
...
-github.com/google/docsy-example github.com/google/docsy@v0.5.1-0.20221017155306-99eacb09ffb0
-github.com/google/docsy-example github.com/google/docsy/dependencies@v0.5.1-0.20221014161617-be5da07ecff1
-github.com/google/docsy/dependencies@v0.5.1-0.20221014161617-be5da07ecff1 github.com/twbs/bootstrap@v4.6.2+incompatible
-github.com/google/docsy/dependencies@v0.5.1-0.20221014161617-be5da07ecff1 github.com/FortAwesome/Font-Awesome@v0.0.0-20220831210243-d3a7818c253f
+github.com/google/docsy-example github.com/google/docsy@v0.6.0
+github.com/google/docsy-example github.com/google/docsy/dependencies@v0.6.0
+github.com/google/docsy/dependencies@v0.6.0
+github.com/twbs/bootstrap@v4.6.2+incompatible
+github.com/google/docsy/dependencies@v0.6.0
+github.com/FortAwesome/Font-Awesome@v0.0.0-20220831210243-d3a7818c253f
```
Comment (deining) [inline README.md:19]: Output does not show the latest version of Font-Awesome, should be:

```
github.com/FortAwesome/Font-Awesome v0.0.0-20221115183454-96cafbd73ec4
```

File: netlify.toml [hunk 3]
```diff
@@ -1,4 +1,4 @@
 [build]
 [build.environment]
-HUGO_VERSION = "0.104.3"
-GO_VERSION = "1.19.2"
+HUGO_VERSION = "0.107.0"
```
Comment (deining) [inline netlify.toml:3]: Latest `hugo` version is `0.109.0` now.

File: netlify.toml [hunk 4]
```diff
@@ -1,4 +1,4 @@
...
+GO_VERSION = "1.19.3"
```
Comment (deining) [inline netlify.toml:4]: Latest `go` version is `1.19.4` now.

**Review by deining — CHANGES_REQUESTED** [review]
I spotted some minor quirks, once these are corrected, PR is ready for merging.

**Comment by deining** [comment]
Deploy on Netlify fails for whatever reason. When trying to check the logs, I'm present with an empty log window, with a whell spinning forever. Looks like the netlify task is stuck and needs restart/maintenance. Is anyone able to fix that?

File: go.mod (hunk 1 above)
Comment (deining) [reply go.mod]: Resolved with commit 3031c33b5e0b.

File: README.md (hunk 2 above)
Comment (deining) [reply README.md]: Resolved with commit 3031c33b5e0b.

File: netlify.toml (hunk 4 above)
Comment (deining) [reply netlify.toml]: Resolved with commit 3031c33b5e0b.

**Comment by deining** [comment]
> Deploy on Netlify fails due to to the use of unsupported Ubuntu 16.04 (Xenial) build image.

With this topic being resolved, I just resolved the outstanding issues raised by me a while ago.

@LisaFC @geriom: can you please have a final look here? Would like to see this merged. We are referring to the this example repo as recommended setup method in the user guide, so this repo should point to the latest official release of docsy. Thanks.

**Comment by chalin** [comment]
@deining - better late than never? Shall we merge this now before updating to 0.7.0?

**Comment by chalin** [comment]
I'm going to go ahead and merge. We can address any required changes in the 0.7.0 PR.

**Comment by chalin** [comment]
> Great, thanks. With this PR merged, we should at least add a tag `v0.6.0` (and maybe even add `v0.6.0` as release). WDYT?

Good idea, thanks. Can take care of that?

**Comment by deining** [comment]
> Good idea, thanks. Can take care of that?

Done.

**Comment by chalin** [comment]
Wonderful! 🎉

**Pull request merged**